app/scripts/models/reliers/oauth.js

```diff
@@ -95,6 +101,23 @@ define([
       return !! this.getSearchParam('code');
     },
 
+    /**
+     * Sets relier properties from the resume token value
+     * @private
+     */
+    _parseResumeToken: function () {
+      var resumeToken = this.getSearchParam('resume');
+      var parsedResumeToken = ResumeToken.parse(resumeToken);
+
+      if (parsedResumeToken) {
+        _.each(RELIER_FIELDS_IN_RESUME_TOKEN, function (itemName) {
+          if (Object.prototype.hasOwnProperty.call(parsedResumeToken, itemName)) {
```

Comment:
`parsedResumeToken` is created from parsing user controlled input, so they could have set `hasOwnProperty` on the value. If they do that it would explode when we attempt to call it as a function.